python/ccxt/async_support/base/exchange.py

```diff
@@ -480,6 +519,48 @@ def format_scientific_notation_ftx(self, n):
 
     # METHODS BELOW THIS LINE ARE TRANSPILED FROM JAVASCRIPT TO PYTHON AND PHP
 
+    def safe_prop(self, obj, key1, defaultValue=None):
+        return self.get_property(obj, key1, defaultValue)
+
+    def safe_prop_2(self, obj, key1, key2, defaultValue=None):
```

Comment:
as I said I don't think we need these methods here if we check these props explicitly

Reply:
we need these methods, as we cant otherwise check these props explicitly in an unified manner. `this.safexxx` does not work with class objects in all langs